specs/CameraCaptureUI/CameraCaptureUI.md

```diff
@@ -207,6 +212,11 @@ and whether or not cropping is allowed by the user interface (UI).
 Provides settings for capturing videos. 
 The settings include format, maximum resolution, maximum duration, and whether or not to allow trimming.
 
+**Important Note**:  
+- `MaxDurationInSeconds` must be a valid value (i.e., the duration should be in the range of 0 to `UINT32_MAX`).
+- `MaxDurationInSeconds` cannot be set if `AllowTrimming` is false.
```

Comment:
Here also may be good to specify behavior if AllowTrimming is false

Reply:
Yes, updated!